core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -654,6 +657,24 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
         .withStatusCode(StatusCodes.OK.intValue)
     )
 
+    for {
```

Comment:
I still hate these shared megamocks. Too much chance for unintended interactions. But that's not the fault of these changes. Oh well.

Reply:
yeah, I hate it too... the expected data for a test is in too many places